rest_lights.cpp

```diff
@@ -3056,17 +3058,17 @@ void DeRestPluginPrivate::handleLightEvent(const Event &e)
                     continue;
                 }
 
-                if (item->lastSet().isValid() && (gwWebSocketNotifyAll || (item->lastChanged().isValid() && item->lastChanged() >= lightNode->lastAttrPush)))
+                if (item->lastSet().isValid() && (gwWebSocketNotifyAll || (item->lastChanged().isValid() && item->needPushChange())))

```

Comment:
As mentioned above, I'd ditch the timestamp checks, here and in further places as well.

```c++
if (item->needPushSet() && (gwWebSocketNotifyAll || item->needPushChange()))
{
}
```

Reply:
I don't think we need to check for `needPushSet()` here?